client/apps/Admin/src/components/CreateClientModal.tsx

```
@@ -1,28 +1,144 @@
+      onSuccess?.();
+      onClose();
+    } catch (error) {
+      const hebrewError = translateError(error);
+      setSubmitError(hebrewError);
+    } finally {
+      setIsSubmitting(false);
+    }
+  }
+
   return (
     <div className="modal-overlay" onClick={onClose}>
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)